cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -50,6 +50,7 @@ const (
 	autoDiscovererTypeASG   = "asg"
 	asgAutoDiscovererKeyTag = "tag"
 	optionsTagsPrefix       = "k8s.io/cluster-autoscaler/node-template/autoscaling-options/"
+	LabelAwsCSITopologyZone = "topology.ebs.csi.aws.com/zone"
```

Comment:
Does this need to be public?  See also failure in linting:

```
/home/runner/work/autoscaler/autoscaler/go/src/k8s.io/autoscaler/cluster-autoscaler/cloudprovider/aws/aws_manager.go:53:2: exported const LabelAwsCSITopologyZone should have comment (or a comment on this block) or be unexported```

Reply:
For this fix, no, it doesn't need to be public. I'll change that.